tasks/html-css-popup.md

```diff
@@ -25,6 +25,40 @@ The goal of this task is to practice coding HTML & CSS from mockup. You are expe
     makes the content of the popup scrollable
 - mobile view is irrelevant for this task
 
+## Recommendations
+- Make sure that your navigation is built by this structure: 
+``` HTML
+  <nav>
```

Comment:
I disagree to provide them with correct solutions, my point is we must give them links to articles or videos with explanations why we recommend this solution.) Because they will use our solutions without understanding how it works and why they use this solution.

Reply:
Updated. Added a link to the article here